spec/controllers/taxi_requests_controller_spec.rb

```diff
@@ -0,0 +1,207 @@
+        end
+      end
+    end
+
+    describe '#create' do
+      def request
+        post :create, params: params
+      end
+
+      let(:params) { { address: address } }
+
+      context '올바른 요청인 경우' do
+        it_behaves_like 'OK 응답 처리', :request
+      end
+
+      context '주소가 너무 긴 경우' do
+        before { params[:address] = 'a' * 101 }
+
+        it_behaves_like 'Bad Request 응답 처리', :request do
+          let(:message) { '주소는 100자 이하로 입력해주세요' }
+        end
+      end
+
+      context '기존 요청이 있는 경우' do
+        before { create(:taxi_request, passenger_id: passenger.id) }
+
+        it_behaves_like 'Conflict 응답 처리', :request do
+          let(:message) { '아직 대기중인 배차 요청이 있습니다' }
+        end
+      end
+    end
+
+    describe '#accept_request' do
+      def request
+        post :accept_request, params: { taxi_request_id: taxi_request.id }
+      end
+
+      let!(:taxi_request) { create(:taxi_request, passenger_id: passenger.id) }
+
+      context '승객이라 요청을 못 하는 경우' do
+        it_behaves_like 'Forbidden 응답 처리', :request do
+          let(:message) { '기사만 배차 요청을 수락할 수 있습니다' }
+        end
+      end
+    end
+  end
+
+  context '유효한 토큰이 있는 기사인 경우' do
+    fixtures :taxi_requests
+
+    let!(:driver) { create(:driver) }
+    let(:payload) { { user_id: driver.id, exp: exp } }
+    let(:token) { JWT.encode(payload, secret_key) }
+
+    before(:each) do
+      driver.token = token
+      driver.save!
+
+      @request.headers["Authorization"] = "Token #{token}"
+    end
+
+    describe '#index' do
+      def request
+        get :index
+      end
+
+      context '올바른 요청인 경우' do
+        it_behaves_like 'OK 응답 처리', :request
+
+        it '응답이 길이 4 이상의 배열이다' do
+          request
+          expect(subject.is_a?(Array)).to eq(true)
+          expect(subject.length >= 4).to eq(true)
+        end
+      end
+    end
+
+    describe '#create' do
+      def request
+        post :create, params: { address: address }
+      end
+
+      context '기사라 요청을 못 하는 경우' do
+        it_behaves_like 'Forbidden 응답 처리', :request do
+          let(:message) { '승객만 배차 요청할 수 있습니다' }
+        end
+      end
+    end
+
+    describe '#accept_request' do
+      def request
+        post :accept_request, params: params
+      end
+
+      let(:params) { { taxi_request_id: taxi_request.id } }
+
+      let!(:passenger) { create(:passenger) }
+      let!(:taxi_request) { create(:taxi_request, passenger_id: passenger.id) }
+
+      context '올바른 요청인 경우' do
+        it_behaves_like 'OK 응답 처리', :request
+      end
+
+      context '존재하지 않는 요청인 경우' do
+        before { params[:taxi_request_id] = 0 }
+
+        it_behaves_like 'Not Found 응답 처리', :request do
+          let(:message) { '존재하지 않는 배차 요청입니다' }
+        end
+      end
+
+      context '이미 수락된 배차 요청인 경우' do
+        let!(:another_driver) { create(:driver) }
+
+        before(:each) do
+          taxi_request.driver = another_driver
+          taxi_request.save!
+        end
```

Comment:
```suggestion
        before(:each) { taxi_request.update!(driver: another_driver) }
```
이렇게 하면 한줄로 코드를 적을 수 있을 것 같아요!

Reply:
[74850f6](https://github.com/travelingaries/taxi-dispatch-backend-api/pull/1/commits/74850f68f0e3f8b9245c94c64557e57124861a7b)에서 제안주신 방법대로 커밋했습니다~